app/models/ncr/work_order.rb

```diff
@@ -203,9 +203,10 @@ def record_changes
       comment_texts = []
       bullet = changed_attributes.length > 1 ? '- ' : ''
       changed_attributes.each do |key, value|
+        former = property_to_s(self.send(key + "_was")) if self.proposal.approved?
```

Comment:
I'd suggest

* Adding a test for the case where it _shouldn't_ be present
* Don't do the `if` inline, so that the body shows up separately in line coverage reports